[2022-07-05 07:21:30,984] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 07:21:31,043] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 07:21:31,043] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:31,043] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:31,043] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:31,102] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 22:00:00+00:00
[2022-07-05 07:21:31,118] {standard_task_runner.py:52} INFO - Started process 586 to run task
[2022-07-05 07:21:31,130] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T22:00:00+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp33fyo2so', '--error-file', '/tmp/tmpzs26enum']
[2022-07-05 07:21:31,135] {standard_task_runner.py:77} INFO - Job 251: Subtask kimuksung_test
[2022-07-05 07:21:31,297] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:31,450] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T22:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T22:00:00+00:00
[2022-07-05 07:21:31,452] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:31,453] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:31,481] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T220000, start_date=20220705T072130, end_date=20220705T072131
[2022-07-05 07:21:31,517] {standard_task_runner.py:92} ERROR - Failed to execute job 251 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:31,553] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:31,624] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:44,850] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 07:25:44,913] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 07:25:44,913] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:44,928] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:44,928] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:44,999] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 22:00:00+00:00
[2022-07-05 07:25:45,027] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T22:00:00+00:00', '--job-id', '282', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpr2x9_520', '--error-file', '/tmp/tmpyo63u5za']
[2022-07-05 07:25:45,028] {standard_task_runner.py:77} INFO - Job 282: Subtask kimuksung_test
[2022-07-05 07:25:45,017] {standard_task_runner.py:52} INFO - Started process 778 to run task
[2022-07-05 07:25:45,405] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:45,785] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T22:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T22:00:00+00:00
[2022-07-05 07:25:45,790] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:45,790] {test.py:22} INFO - pubg
[2022-07-05 07:25:45,790] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:45,907] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T220000, start_date=20220705T072544, end_date=20220705T072545
[2022-07-05 07:25:46,054] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:46,227] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:18,396] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 08:06:18,454] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 08:06:18,454] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:18,454] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:18,454] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:18,536] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 22:00:00+00:00
[2022-07-05 08:06:18,545] {standard_task_runner.py:52} INFO - Started process 2162 to run task
[2022-07-05 08:06:18,561] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T22:00:00+00:00', '--job-id', '333', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpi3ampomn', '--error-file', '/tmp/tmpaoqc13jh']
[2022-07-05 08:06:18,605] {standard_task_runner.py:77} INFO - Job 333: Subtask kimuksung_test
[2022-07-05 08:06:19,347] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:20,563] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T22:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T22:00:00+00:00
[2022-07-05 08:06:20,567] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:20,568] {test.py:22} INFO - pubg
[2022-07-05 08:06:20,570] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:20,747] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T220000, start_date=20220705T080618, end_date=20220705T080620
[2022-07-05 08:06:21,152] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:21,366] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:29,619] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 08:11:29,666] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [queued]>
[2022-07-05 08:11:29,667] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:29,667] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:29,667] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:29,732] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 22:00:00+00:00
[2022-07-05 08:11:29,758] {standard_task_runner.py:52} INFO - Started process 2466 to run task
[2022-07-05 08:11:29,779] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T22:00:00+00:00', '--job-id', '400', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpjh_iw_8r', '--error-file', '/tmp/tmp0ihlo3li']
[2022-07-05 08:11:29,779] {standard_task_runner.py:77} INFO - Job 400: Subtask kimuksung_test
[2022-07-05 08:11:30,111] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T22:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:30,541] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T22:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T22:00:00+00:00
[2022-07-05 08:11:30,544] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:30,545] {test.py:22} INFO - pubg
[2022-07-05 08:11:30,546] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:30,682] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T220000, start_date=20220705T081129, end_date=20220705T081130
[2022-07-05 08:11:30,767] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:30,875] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
